identity-enabler/holder-mobile-app/src/pages/CreatePresentation.svelte

```diff
@@ -78,8 +80,16 @@
         showTutorial = true;
     }
 
-    function onClickJSON() {
-        showJSON = true;
+    async function onClickDataMatrix() {
```

Comment:
can we just use the double click event https://developer.mozilla.org/en-US/docs/Web/API/Element/dblclick_event

Reply:
The compat matrix on that link shows Android WebView as unsupported